### PR TITLE
Add ENA_template_ERC000011_sample.xml

### PR DESCRIPTION
**Pull request by rabuono**
Based on [ENA_template_vir_sample.xml](https://github.com/usegalaxy-eu/ena-upload-cli/blob/159475723974f4b85f22574866701055ff0a7984/ena_upload/templates/ENA_template_vir_sample.xml) and ENA's Sample checklist ERC000011, made ENA_template_ERC000011_sample.xml

### CODE REVIEW

**Comment by rabuono** [comment]
Made a table [like this one](https://github.com/usegalaxy-eu/ena-upload-cli#supported-columns-for-viral-sample-submissions) for the template.
[ERC000011_for_bert.xlsx](https://github.com/usegalaxy-eu/ena-upload-cli/files/6887203/ERC000011_for_bert.xlsx)

File: ena_upload/templates/ENA_template_ERC000011_sample.xml [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+    xsi:noNamespaceSchemaLocation="ftp://ftp.sra.ebi.ac.uk/meta/xsd/sra_1_6/SRA.sample.xsd">
+    <py:for each="index, row in df.iterrows()">
+    <SAMPLE alias="${row.alias}" center_name="${center}">
+        <!--TODO: UNIQUE NAME FOR SAMPLE-->
+        <!--TODO: CENTER NAME-->
+        <TITLE>${row.title}</TITLE>
+        <!--TODO: A SHORT INFORMATIVE DESCRIPTION OF THE SAMPLE-->
+        <SAMPLE_NAME>
+            <TAXON_ID>${row.taxon_id}</TAXON_ID>
+            <!--TODO: PROVIDE NCBI TAXID FOR ORGANISM (e.g. 9606 for human)-->
```
Comment (bgruening) [inline ena_upload/templates/ENA_template_ERC000011_sample.xml:20]: @rabuono are you planning to address those TODOs before we merge?

**Review by bgruening — APPROVED** [review]
Thanks a lot!

File: ena_upload/templates/ENA_template_ERC000011_sample.xml (hunk 1 above)
Comment (rabuono) [reply ena_upload/templates/ENA_template_ERC000011_sample.xml:20]: Those TODOs were already in the original [ENA_template_vir_sample.xml](https://github.com/usegalaxy-eu/ena-upload-cli/blob/159475723974f4b85f22574866701055ff0a7984/ena_upload/templates/ENA_template_vir_sample.xml) from which I based this new one on. I don't know what was the original plan on how to address them.
So, in short, no.

**Comment by bedroesb** [comment]
@bgruening I am working on a solution so the user can switch between other checklists better

**Comment by bedroesb** [comment]
Will merge this PR and add improvements with another PR.

**Pull request merged**